gini/gini-vision-lib-ios
# Feature tooltip reorder pages

When the multipage review screen appears for the first time, a tooltip explaining the reorder button should be shown.
It has been added also the option to reset the user defaults in the example app.

### How to test
Run the Example app, take a picture, go to the multipage screen and see the tooltip.

### Merging
Automatic
